app/translations/cy/census_individual_gb_wls_cy.po

```diff
@@ -754,10 +753,10 @@ msgid "What was <em>{person_name_possessive}</em> full job title?"
 msgstr "Beth oedd teitl llawn swydd <em>{person_name_possessive}</em>?"
 
 msgid "Briefly describe what <em>{person_name}</em> does in their main job."
-msgstr "Beth mae <em>{person_name}</em> yn ei wneud yn y brif swydd."
+msgstr "Beth mae <em>{person_name}</em> yn ei wneud yn y brif swydd?"
 
 msgid "Briefly describe what <em>{person_name}</em> did in their main job."
-msgstr "Beth oedd <em>{person_name}</em> yn ei wneud yn y brif swydd."
+msgstr "Beth oedd  yn ei wneud yn y brif swydd?"
```

Comment:
This is missing the placeholder...

Reply:
Also that space between `oedd` and `yn` is actually two spaces

Reply:
Fixed